app/src/main/java/com/example/randomusergenerator/data/remote/dto/UserGeneratorResponse.kt

```diff
@@ -1,56 +1,56 @@
 package com.example.randomusergenerator.data.remote.dto
 
 data class UserGeneratorResponse(
-    val result: List<Result>,
-    val info: Info
+    val result: List<Result>? = emptyList(),
+    val info: Info? = Info()
 )
 
 
 data class Result(
-    val gender: String,
-    val name: Name,
-    val email: String,
-    val location: Location,
-    val login: LoginDetails,
-    val dob: DateOfBirth,
-    val picture: PictureData,
+    val gender: String? = "",
```

Comment:
Make the empty strings nullable because some data might return `null` instead of `""`.

Reply:
I thought that assigning a value was just a default value that should be used when null is returned